content/getting-started/white-label-apps.md

```diff
@@ -0,0 +1,96 @@
+---
+description: How to white label apps with Codemagic
+title: White label apps
+weight: 10
+---
+
+White labeling is the practice of building multiple versions of the same application. This can range from one or two up to thousands of different versions. 
+
+This documentation page should be considered as a minimum viable page to help answer most common questions developers have specifically about setting up CI/CD for their white label product in order to understand if they can achieve their desired use case with Codemagic.
+
+The documentation is open-sourced and we encourage you to provide feedback and contributions in the documentation GitHub repository [here](https://github.com/codemagic-ci-cd/codemagic-docs). If you are looking for general documentation about Codemagic please use the search function or browse other parts of the documentation. 
+
+## How to setup Codemagic with ~10 different versions
+
+If you have around 10 different versions of your application that you are trying to set up CI/CD for then consider separating each version into a unique workflow. 
```

Comment:
```suggestion
If you have around 10 different versions of your application that you are trying to set up CI/CD for, consider separating each version into a unique workflow. 
```

Reply:
I would add here that separating the versions into different workflows allows using different build schemas, code signing settings and publishing destinations, etc. Perhaps it would make sense to also mention the option to trigger the workflows based on changes in certain files. That's basically everything that allows automating the builds of different flavors.

Reply:
Added some more details.